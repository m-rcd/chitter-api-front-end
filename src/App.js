import React from 'react';
import './App.css';
import Peep from './components/peeps';
import NewPeep from './components/new_peep';

export default class App extends React.Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <p> Chitter </p>
        </header>
        <div>
          <NewPeep />
        </div>
        <div>
          <Peep />
        </div>
      </div>
    );
  }
}
