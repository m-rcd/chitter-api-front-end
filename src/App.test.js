import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

it('App Snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
