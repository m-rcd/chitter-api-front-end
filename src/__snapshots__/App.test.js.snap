// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <p>
       Chitter 
    </p>
  </header>
  <div>
    <form>
      <input
        name="peep"
        type="text"
      />
      <input
        type="submit"
        value="Submit"
      />
    </form>
  </div>
  <div>
    <div
      className="peeps"
    />
  </div>
</div>
`;
